src/main/java/org/libtorrent4j/Vectors.java

```diff
@@ -137,4 +138,35 @@ public static byte_vector ascii2byte_vector(String s) {
 
         return arr;
     }
+
+    public static byte_array_32 bytes2byte_array_32(byte[] arr) {
+        byte_array_32 v = new byte_array_32();
+
+        for (int i = 0; i < 32; i++) {
+            v.set(i, arr[i]);
+        }
+
+        return v;
+    }
+
+    public static byte[] byte_array2bytes(byte_array_64 v) {
```

Comment:
Codacy found an issue: [The static method name 'byte_array2bytes' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/aldenml/libtorrent4j/pullRequest?prid=5867294)